Application.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+public class Application {
+    public static void main(String[] args) {
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());

```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes
use for loop, only one lottery instance is needed
when you find yourself naming variables with numbers - time to check if you doing everything right